packages/admin-panel/src/library.js

```diff
@@ -0,0 +1,8 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import 'react-table/react-table.css';
```

Comment:
Don't quite understand the reason for this import? Does it re-style all the components that are being exported here?

Reply:
It literally just adds the react-table stylesheet to the bundle. It doesn't actually style the components per say, it's just a bunch of old school css selectors that apply to the whole page. Not sure if that answers your question? Without it there are no default react table styles.